### PR TITLE
news fetch program in python

### PR DESCRIPTION
**Pull request by Roshaen**
## Assigned Issue Number ? { if does not exist then create one }
#007

## Language Opted ?
Python

## Number of files added or updated ?
2

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've referred the correct issue number.
- [x] I've fill up this entire template correctly.

### CODE REVIEW

**Comment by Roshaen** [comment]
@Arsenic-ATG Please review this

**Comment by Arsenic-ATG** [comment]
Will review it by tomorrow 👍

**Review by Arsenic-ATG — APPROVED** [review]
functionally working on windows operating system. 👍

It can also be easily be made cross platform as the OS specific library is only used to speak out the news and does not do contribute to core functionality of the program so can easily be removed/replaced.

**Comment by Arsenic-ATG** [comment]
@Roshaen thanks again for your contribution 🏅

File: Automations/newsFetch-python/main.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import requests
+from win32com.client import Dispatch
```
[the remark this answers is not in the document]
Comment (Roshaen) [reply Automations/newsFetch-python/main.py:2]: @Arsenic-ATG yes I think Pyttsx would work well
I'll make an another PR 

THANK YOU for the suggestion 🤗

**Pull request merged**